cypress/integration/rules.spec.js

```diff
@@ -0,0 +1,27 @@
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Rules page', () => {
+  it('when not logged in it should redirect to login page', () => {
+    cy.visit('#/rules');
+    shouldBeRedirectedToLogin();
+  });
+
+  describe('when logged in', () => {
+    beforeEach(() => {
+      cy.login();
+    });
+
+    it('should display a link to view rules', () => {
+      cy.visit('/');
+      cy.get('nav').contains('Rules').should('exist');
+    });
+
+    it('should display a list of rules', () => {
+      cy.visit('/');
+      cy.get('nav').contains('Rules').click();
+      cy.url().should('include', '/#/rules');
+      cy.get('table tbody tr').should('have.length', 1);
+      cy.get('table tr[data-value="MOD09GQ_TEST_kinesisRule"]').should('exist');
+    });
+  });
```

Comment:
Can we have one more test in this section that you can click on a rule and see the specific page and all the correct data for that rule?

Reply:
This has been added starting on line 40